### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by palson-cf**

### CODE REVIEW

File: docs/resources/project.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Project Resource
+
+The top-level concept in Codefresh. You can create projects to group pipelines that are related. In most cases a single project will be a single application (that itself contains many micro-services). You are free to use projects as you see fit. For example, you could create a project for a specific Kubernetes cluster or a specific team/department.
+
```
Comment (sharon-codefresh) [inline docs/resources/project.md:4]: Please consider to add:
https://codefresh.io/docs/docs/configure-ci-cd-pipeline/pipelines/#pipeline-concepts

File: examples/README.md [hunk 2]
```diff
@@ -1,76 +1,78 @@
 # Example
```
Comment (sharon-codefresh) [inline examples/README.md:2]: Please consider adding a few words what this example create

File: docs/resources/project.md (hunk 1 above)
Comment (palson-cf) [reply docs/resources/project.md:5]: according to the [docs[(https://www.terraform.io/docs/registry/providers/docs.html#resource-data-source-headers) the description should contain a few words about what resource does and links to the official documentation.
Adde link to the docs.

File: examples/README.md (hunk 2 above)
Comment (palson-cf) [reply examples/README.md:2]: done

**Pull request merged**